wiki/en/en-Tips-Tricks-More.md

```diff
@@ -149,3 +150,43 @@ Now you can run these with the `service start` command, for example:
 `sudo service jamulusTogglerec start` (assuming you named your unit file `jamulusTogglerec.service`)
 
 You can see the result of these commands if you run `service jamulus status`, or by viewing the logs.
+
+## Quality of Service
+
+Jamulus uses DSCP/CS4 to opportunistically deal with bufferbloat.<br>
+DSCP/CS4 is 128 (or 0x80) and is compatible with IPv4 and IPv6.<br>
+Other values can be set with: jamulus --qos `[0..255]`<br>
+( please see the informative RFC4594 and jamulus --help )<br>
+To disable QoS use: --qos 0 
+
+#### Windows Quality of Service instructions
+
+Prior to Windows Vista / Server 2008, Windows allowed software developers to set the
+Quality of Service field on IP traffic.
+With the release of Vista / Server 2008, Microsoft began to overwrite DSCP values
+that were set by applications with the Policy-based QoS layer.
+
+In order to enable Quality of Service you must follow these instructions:
+
+In Search box beside Start menu Type: Local Group Policy Editor (enter)<br>
+In new window, (click) on the menu icon to display the Actions third panel<br>
+Looking at the first panel of the Local Group Policy Editor<br>
+&nbsp;Local Computer Policy<br>
```

Comment:
Again, the formatting here looks questionable.

Reply:
I formatted the text so that it looked reasonable in preview mode; this will give you an indication of what I was trying to achieve. I'm not an expert at markdown, xml, or html. I also figured that the text may not meet everyones approval. Be free to change things.